velox/type/Variant.h

```diff
@@ -105,6 +105,20 @@ struct VariantTypeTraits<TypeKind::ARRAY> {
   using stored_type = std::vector<variant>;
 };
 
+template <>
+struct VariantTypeTraits<TypeKind::SHORT_DECIMAL> {
+  using native_type = typename TypeTraits<TypeKind::SHORT_DECIMAL>::NativeType;
+  // @TODO: This just a place holder. Replace with ShortDecimalVariant
```

Comment:
Let's replace the body of this function with VELOX_NYI().

Reply:
This is a struct, so not able call VELOX_NYI here. Will leave it as it is for now.